pkgs/top-level/python-packages.nix

```diff
@@ -8751,6 +8751,8 @@ in {
     };
   };
 
+  pepper = callPackage ../development/python-modules/pepper { };
```

Comment:
@FRidh That said, it is a bit weird to have  `ansible` in a complete different place ...